### DISCUSSION TITLE
인용 방법

### DISCUSSION THREAD

**Discussion by Woon-2** [MyeongWoonJang/cpp-study#167] [General]
## Reference 표기

웹문서를 인용하는 경우 글의 맨 뒤에 `Reference` 헤더를 두어 다음의 양식을 따라 인용합니다.   
[웹사이트 이름] 웹문서 제목

논문이나 단행본을 인용하는 경우 글의 맨 뒤에 `Reference` 헤더를 두어 다음의 양식을 따라 인용합니다.   
저자명. 『서명』. 번역자 옮김. 발행지:발행처, 발행년.

>## Reference
>
> Thaler, Richard H., and Cass R. Sunstein. 『넛지: 똑똑한 선택을 이끄는 힘』. 안진환 옮김. 서울: 웅진씽크빅, 2009.

## 본문에서의 짧은 인용

본문에서 `Reference`에 포함한 웹문서를 짧게 인용할 때에는    
인용한 부분을 클릭하면 웹문서로 연결될 수 있도록 링크화합니다.    
본문에서 `Reference`에 포함한 논문/단행본을 짧게 인용할 때에는 ()안에 인용한 자료의 저자와 발행년, 페이지를 표기합니다.

## 본문에서의 긴 인용

본문에서 긴 내용을 인용할 때에는,   
인용문 블록을 만들어 내용을 그대로 인용한 뒤, 출처를 표시합니다.

> 이와 같이 인용문 블록을 만들어 내용을 그대로 인용하고,
> source: [웹사이트 이름] 웹문서 제목 또는
> source:  저자명. 『서명』. 번역자 옮김. 발행지:발행처, 발행년.

단, 필요한 내용들이 흩어져 있어 중간에 내용을 생략하는 경우,   
두 개의 빈 줄 사이에 `...`를 한 문장으로 작성합니다.

> 앞부분의 내용
>
> ...
>
> 뒷부분의 내용

**Comment by Lcat12345** [comment]
감사합니다
